gulpfile.cjs

```diff
@@ -383,6 +383,33 @@ function combineRelease() {
 
 gulp.task("combineRelease", gulp.series("build", combineRelease));
 
+// Downloads Draco3D files from gstatic servers
+async function downloadDraco() {
+  const wrapperPromise = new Promise(function (resolve) {
```

Comment:
Should there be error handling here?

Reply:
Yes something like

```js
  const wrapperPromise = new Promise(function (resolve, reject) {
    request(...)
      .pipe(...)
      .on("error", reject);
      .on("finish", resolve);
```

Also make this a standalone function that takes a url, a file path, and returns a promise. It'll keep the `downloadDraco` code simpler.